service/rpccommon/server.go

```diff
@@ -89,7 +89,7 @@ func (s *ServerImpl) Stop(kill bool) error {
 }
 
 // Restart restarts the debugger.
-func (s *ServerImpl) Restart() error {
+func (s *ServerImpl) Restart(args []string) error {
```

Comment:
There's no point in adding this argument here since you are not using it below.

Reply:
Ok, reverted.